jest.config.js

```diff
@@ -1,30 +1,30 @@
 // jest.config.js
 module.exports = {
   collectCoverageFrom: [
-    "**/*.{js,jsx,ts,tsx}",
-    "!**/*.d.ts",
-    "!**/node_modules/**",
+    '**/*.{js,jsx,ts,tsx}',
```

Comment:
Was this an auto-update from prettier?

Reply:
Yes

Reply:
:( I prefer double quotes but why did this happen if the configs were not updated? Also, in `src/__tests__/setup.ts` some strings are double quoted but the new addition is single quoted...

Reply:
Is issue is just a personal annoyance. When I save the file locally, my local editor's linting rules updates this to double quotes bu then eslint complains. I'll create another ticket for this, it's not a blocker for this release.